internal/kafka/internal/workers/cluster_mgrs/clusters_mgr.go

```diff
@@ -881,6 +881,10 @@ func (c *ClusterManager) buildObservatoriumDexSecretResource() *k8sCoreV1.Secret
 
 func (c *ClusterManager) buildObservabilityRemoteWriteServiceAccountCredential(cluster *api.Cluster) *k8sCoreV1.Secret {
 	return &k8sCoreV1.Secret{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: metav1.SchemeGroupVersion.Version,
```

Comment:
I am not sure this is correct.
The APIVersion of a Secret is `v1` (or `core/v1`), not `meta.k8s.io/v1` which is what is being referenced there

Reply:
This should be set to `k8sCoreV1.SchemeGroupVersion.String()` instead

Reply:
This seems to be the `APIVersion` used for secrets throughout the application.

Reply:
See K8s Secret API doc: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.26/#secret-v1-core

Reply:
For this change, would it be fine to reference the correct group version and deal with the others in a separate issue/PR?

Reply:
> For this change, would it be fine to reference the correct group version and deal with the others in a separate issue/PR?

Yes. I will take care of fixing it for the others